components/checkout/payments/DummyCreditCardSection.tsx

```diff
@@ -43,17 +44,25 @@ export const DummyCreditCardSection = ({
     total: formatPrice(totalPrice),
   });
 
+  let defaultValues = DEMO_MODE
```

Comment:
```suggestion
  const defaultValues = DEMO_MODE
```
I don't see them being reassigned